docs/user/aws/install_upi.md

```diff
@@ -0,0 +1,432 @@
+# Install: User Provided Infrastructure (UPI)
+
+The steps for performing a UPI-based install are outlined here. Several [CloudFormation][cloudformation] templates are
+provided to assist in completing these steps or to help model your own.
+
+## Create Ignition Configs
+
+The machines will be started manually. Therefore, it is required to generate the bootstrap and machine Ignition configs
+and store them for later steps.
+
+```console
+$ openshift-install-linux-amd64 create ignition-configs
+? SSH Public Key /home/user_id/.ssh/id_rsa.pub
+? Platform aws
+? Region us-east-1
+? Base Domain example.com
+? Cluster Name openshift
+? Pull Secret [? for help]
+```
+
+After running the command, several files will be available in the directory.
+
+```console
+$ tree
+   .
+   ├── auth
+   │   └── kubeconfig
+   ├── bootstrap.ign
+   ├── master.ign
+   ├── metadata.json
+   └── worker.ign
+```
+
+## Create/Identify the VPC to be Used
+
+You may create a VPC with various desirable characteristics for your situation (VPN, route tables, etc.). The recommended
```

Comment:
nit: drop `recommended` for UPI we are not providing any recommendations, only examples.

the goal for UPI to tell users what is needed and this is how you can create what is needed... the latter would mean that we are on the hook to maintain these as deliverables.

Reply:
Fixed.